src/components/OwnerList.tsx

```diff
@@ -0,0 +1,82 @@
+import { useEffect, useState } from 'react';
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  Paper,
+  TablePagination,
+} from '@material-ui/core';
+import { findAllOwners } from '../service/PetClinicService';
+
+const Owners = () => {
+  const [owners, setOwners] = useState([
+    { id: '', firstName: '', lastName: '', address: '', city: '', phone: '' },
+  ]);
+
+  useEffect(() => {
+    findAllOwners()
+      .then((data) => {
+        setOwners(data);
+      })
+      .catch((error) => {
+        console.error(error);
+      });
+  }, []);
+
+  const [page, setPage] = useState(0);
+  const [rowsPerPage, setRowsPerPage] = useState(8);
+
+  const handleChangePage = (newPage) => {
+    setPage(newPage);
+  };
+
+  const handleChangeRowsPerPage = (event: React.ChangeEvent<any>) => {
+    setRowsPerPage(parseInt(event.target.value, 10));
+    setPage(0);
+  };
+
+  return (
+    <div>
+      <TableContainer component={Paper}>
+        <Table>
+          <TableHead>
+            <TableRow>
+              <TableCell>Name</TableCell>
+              <TableCell>Address</TableCell>
+              <TableCell>City</TableCell>
+              <TableCell>Phone</TableCell>
+            </TableRow>
+          </TableHead>
+          <TableBody>
+            {owners
+              .slice(page * rowsPerPage, page * rowsPerPage + rowsPerPage)
+              .map((owner) => (
+                <TableRow key={owner.id}>
+                  <TableCell>
+                    {owner.firstName} {owner.lastName}
```

Comment:
This looks not safe! 

Consider using [string literal](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) or concatenate with a space in the middle.

Reply:
ok I've done this:
```
{owner.firstName + " " +owner.lastName}
```
I didn't understand why was it not safe before?

Reply:
The fact you use two variables as strings, inside JSX structure without string literals and a space inside, this may lead to mistakes like : 
```jsx
<TableCell>
{owner.firstName} 
{owner.lastName}
``` 
👉 someone may go back to the line without noticing the space!